alembic/versions/e951d7a7e751_add_stats_tvl.py

```diff
@@ -0,0 +1,24 @@
+"""add stats_tvl
```

Comment:
The `alembic merge heads` should be run after you merge main into this branch, otherwise it will generate just an empty revision because there are no revision branches to be merged

Reply:
hmmm weird, I just merged `main` ad run the merge command again but it's still generating empty revisions?